content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+
+### Facet Panel
+
+Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
+{{< img src="log/explore/facet_panel.png" alt="Facet panel" >}}
+
+### Expanded view
+
+Expand a facet to access advanced search options like 
+Including or excluding a pattern within a facet.
+
+For example: You want to have all host starting by “i-06”: 
+{{< img src="log/explore/expanded_view.png" alt="Expanded view" style="width:75%;" >}}
+
+## Search bar
+
+The search query language is based on the Lucene query string:
+
+[Apache Lucene - Query Parser Syntax](http://lucene.apache.org/core/2_9_4/queryparsersyntax.html)
+
+All search parameters are contained in the url, so it is very simple to share your view.
+
+### Search syntax
+A query is broken up into terms and operators.
+
+There are two types of terms:
+
+* A **Single Term** is a single word such as "test" or "hello".
+
+* A **Sequence** is a group of words surrounded by double quotes such as "hello dolly" or "hello dolly"~2.
+
+Multiple terms can be combined together with Boolean operators to form a more complex query.
+
+Here are the available boolean operators:
+
+||||
+|:----|:----|:----|
+| **Operator** | **Description ** | **Example **|
+| `AND` | **Intersection**: both terms are in the selected events (if nothing is added, AND is taken by default) | authentication AND failure |
+| `OR` | **Union**: either terms is contained in the selected events| authentication OR password|
+| `-` | **Exclusion**: the following term is NOT in the event |authentication AND -password|
+
+### Facet search 
+To search on specific [facet](#facets) you need to [add them as facet first]()
+then add `@` to specify you are searcing on a facet.
+
+For instance, if your facet name is service and you want to filter on the nginx value just enter: 
+
+`@service:nginx`
+
+### Wildcards
+To perform a multi-character wildcard search, use the `*` symbol as follows:
+
+* `service:web*`  matches every log message that have an service starting by “web”.
+* `hello*` matches all log message starting with hello
+* `*hello` matches all log message that ends with hello
+
+### Tags
+
+Your logs inherit tags from their [host](https://docs.datadoghq.com/hostnames/) and [integrations](https://docs.datadoghq.com/integrations/). 
+They can be used in the search and in facets as well:
+
+* `test` is searching for the tag #test.
+* `("env:prod" OR test)` matches all logs with the tag #env:prod or the tag #test 
+* `("service:srvA" OR "service:srvB")` or `(service:(srvA OR srvB))` Matches all logs that contains tags #service:srvA or #service:srvB.
+* `("env:prod" AND -”version:beta”)` matches all logs that contains #env:prod and that do not contains #version:beta
+
+### Autocomplete
+Typing a complex query can sometime be a bit cumbersome, use the search bar autocomplete feature to complete your query with existing data values:
```

Comment:
"Typing a complex query can be cumbersome. Use the search bar's autocomplete feature to complete your query using existing values."

Reply:
👍